### DISCUSSION TITLE
Add auto fill to github.com/search page

### DISCUSSION THREAD

**Discussion by 0x4248** [community/community#63461] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I would like to have the code search auto fill like in the other pages of GitHub when you press `/`

Examples of the autofill:
![IMG_2937](https://github.com/community/community/assets/60709927/896c2914-b72d-4e48-aa29-1bbe1eecebcf)
![IMG_2938](https://github.com/community/community/assets/60709927/e2e407b5-4a10-4e7e-9145-e1dc346f14ff)

Examples of /search page that dont have this

![IMG_2934](https://github.com/community/community/assets/60709927/e9726d6c-5729-4b81-a508-4814bff7459a)
![IMG_2935](https://github.com/community/community/assets/60709927/d1e20372-9092-4800-bac6-9bf50d912574)

I was also thinking of a dropdown menu next to the search bar saying repository’s, code, issues, wiki, users, commits. (ignore the bad drawing)

![IMG_2936](https://github.com/community/community/assets/60709927/34a560ff-000f-46f8-8405-41cf7695c340)

Like this but next to the search bar

![IMG_2939](https://github.com/community/community/assets/60709927/674f7a4c-b3b7-4d2a-b2ac-a0ca532eefb6)

**Comment by look** [comment, marked as answer]
Thanks for the feedback! We plan to add autocomplete to this page, but haven't been able to get it done yet. It's on our roadmap.
